data/arabic/recommendations/index.json

```diff
@@ -0,0 +1,177 @@
+[
+  {
+    "headlines":{
+      "shortHeadline":"الأمم المتحدة: جرائم حرب \"فظيعة\" ارتكبت في معركة ادلب",
+      "headline":"الصراع في سوريا: الأمم المتحدة تقول إن جرائم حرب \"فظيعة\" ارتكبت في معركة ادلب"
+    },
+    "locators":{
+      "assetUri":"\/arabic\/middleeast-53329495",
+      "cpsUrn":"urn:bbc:content:assetUri:arabic\/middleeast-53329495",
+      "curie":"http:\/\/www.bbc.co.uk\/asset\/30b4d0f3-cd56-4bd6-a5d5-ceca756ed099",
+      "assetId":"53329495"
+    },
+    "summary":"الأمم المتحدة تقول إن المدنيين السوريين قاسوا \"معاناة غير معقولة\" على أيدي القوات السورية والمسلحين.",
+    "timestamp":1594187148000,
+    "language":"ar",
+    "passport":{
+      "category":{
+        "categoryId":"http:\/\/www.bbc.co.uk\/ontologies\/applicationlogic-news\/News",
+        "categoryName":"News"
+      },
+      "campaigns":[
+        {
+          "campaignId":"5a988e4739461b000e9dabfc",
+          "campaignName":"WS - Update me"
+        }
+      ],
+      "taggings":[
+
+      ]
+    },
+    "indexImage":{
+      "id":"113279847",
+      "subType":"index",
+      "href":"http:\/\/c.files.bbci.co.uk\/12491\/production\/_113279847_1.jpg",
+      "path":"\/cpsprodpb\/12491\/production\/_113279847_1.jpg",
+      "height":371,
+      "width":660,
+      "altText":"فتى سوري مصاب يجلس داخل سيارة بعد غارة جوية حكومية في معرة مصرين، شمال غرب سوريا، في 25 فبراير\/شباط 2020",
+      "copyrightHolder":"AFP",
+      "type":"image"
+    },
+    "id":"urn:bbc:ares::asset:arabic\/middleeast-53329495",
+    "type":"cps"
+  },
+  {
+    "headlines":{
+      "shortHeadline":"فرنسا: تركيا تمارس لعبة خطرة في ليبيا لن نتسامح معها",
+      "headline":"الحرب في ليبيا: الرئيس الفرنسي ماكرون يقول إن تركيا تمارس \"لعبة خطرة\" في ليبيا"
+    },
+    "locators":{
+      "assetUri":"\/arabic\/middleeast-53146050",
```

Comment:
any idea why the links and texts in this file are escaped with trailing slashes ? the ones in the `mundo` file don't seem to be.

Reply:
Looks like something weird happened when I copied and pasted. Will amend, although some of the text does have trailing slashes in the data we get from ares.